indico/modules/designer/client/js/index.js

```diff
@@ -33,6 +33,12 @@
 
   var DEFAULT_PIXEL_CM = 50;
 
+  var imageTypes = ['ticket_qr_code', 'event_logo'];
```

Comment:
I don't think we still need this hardcoded list. Also, use `let` so you can reassign later; `var` should not be used in any new code.

Reply:
OK, i'll put `let`. I would like to keep it just as a default...

Reply:
I disagree with keeping a default that is never used anyway... if the code never uses the array I'd just default to `null` so if something does end up using it too early, it breaks instead of silently using different data..